File: Chronos.App/ClientApp/src/app/models/rules/story-rules.ts
import { Variable } from "./variable";

export class StoryRules {
    title = '';
    description = '';
    acceptanceCriteria = '';
    titleVariables: Variable[] = [];
    tags: string[] = [];
    defaultTags: string[] = [];

    constructor(obj?: Partial<StoryRules>) {
        if (obj)
            Object.assign(this, obj);
    }
}

File: Chronos.Domain/Entities/FeatureRules/StoryRules.cs
using System.Runtime.Serialization;

namespace Chronos.Domain.Entities.FeatureRules
{
    [DataContract]
    public class StoryRules
    {
        [DataMember]
        public string Title { get; set; }

        public TitleVariable[] TitleVariables { get; set; }

        [DataMember]
        public string Description { get; set; }

        [DataMember]
        public string AcceptanceCriteria { get; set; }

        [DataMember]
        public string[] Tags { get; set; }

        public string[] DefaultTags { get; set; }
    }
}
